.gitignore

```diff
@@ -12,6 +12,7 @@ tmp/
 
 # macOS
 .DS_Store
+.DS-Store
```

Comment:
To my knowlegde, `.DS-Store` isn't a legit macOS name, so I've vote against adding this here:

```suggestion
```

Besides, the files got committed anyhow! 😄 
They need to be removed from the commit.

Reply:
For the record: I was unable to resolve this issue, that is, [figure out why my macOS version does that](https://x.com/jkowall/status/1726428762965721302?s=20), so, accepting it tho I still don't know what's going on ha ha